lib/licenses.nix

```diff
@@ -545,6 +545,13 @@ lib.mapAttrs mkLicense ({
     free = false;
   };
 
+  firefox = {
+    fullName = "Firefox Terms of Use";
+    url = "https://www.mozilla.org/about/legal/terms/firefox/";
+    free = false;
+    redistributable = false;
```

Comment:
Given that the terms don't mention redistribution, and that the free licenses still apply, I don't see why the additional terms would suddenly make it non-redistributable.

Reply:
I'm not entirely sure that MPL-2.0 _does_ still apply. §3.2b:

> You may distribute such Executable Form under the terms of this License, or sublicense it under different terms, provided that the license for the Executable Form does not attempt to limit or alter the recipients’ rights in the Source Code Form under this License.

so, by my interpretation, Mozilla are exercising the "sublicense it [the binary] under different terms" clause, and it's at best unclear whether they still intend MPL-2.0 to apply to the binaries they distribute.

Reply:
Do these binaries still say the following in about:licenses?

> Binaries of this product have been made available to you by the [Mozilla Project](http://www.mozilla.org/) under the Mozilla Public License 2.0 (MPL). [Know your rights](about:rights).

Reply:
Good point! [The file containing that text](https://github.com/mozilla/gecko-dev/blob/63ee2f21e89042133837bff57786a768462d696a/browser/base/content/overrides/app-license.html) has not been touched in nearly 2 years, so I would _presume_ so, but I haven't downloaded a recent binary to check.

The binaries definitely at least used to be MPL-2.0: https://bugzilla.mozilla.org/show_bug.cgi?id=464994#c15

Reply:
Updated to `redistributable = true`.